xml/System.Reflection.Metadata/ImportScope.xml

```diff
@@ -23,8 +23,16 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides information about the lexical scope within which a group of imports are available. This information is stored in debug metadata.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+See [ImportScope Tab;e" 0x35](https://github.com/dotnet/corefx/blob/master/src/System.Reflection.Metadata/specs/PortablePdb-Metadata.md#importscope-table-0x35) for more information.
```

Comment:
Found a typo, @rpetrusha, which I'm correcting. After the build finishes, if there are no warnings, we can merge:

```suggestion
See [ImportScope Table" 0x35](https://github.com/dotnet/corefx/blob/master/src/System.Reflection.Metadata/specs/PortablePdb-Metadata.md#importscope-table-0x35) for more information.
```